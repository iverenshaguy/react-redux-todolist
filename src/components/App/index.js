import React from 'react';
import './App.css';
import Todos from '../Todos';
import todosApi from '../../helpers/api/todos.api';

const App = () => (
  <div>
    <Todos api={todosApi} />
  </div>
);

export default App;
