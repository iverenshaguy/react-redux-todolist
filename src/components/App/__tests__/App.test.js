import App from '../index';

it('renders correctly', () => {
  const wrapper = shallow(<App />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
