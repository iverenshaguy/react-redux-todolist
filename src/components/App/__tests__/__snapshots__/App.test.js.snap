// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <Todos
    api={
      Object {
        "createTodo": [Function],
        "deleteTodo": [Function],
        "getTodos": [Function],
        "updateTodo": [Function],
      }
    }
  />
</div>
`;
